app/controllers/application.js

```diff
@@ -0,0 +1,41 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+	isExpanded: true,
+	isNotLogin: Ember.computed('this.currentPath', function() {
+	  	if (this.currentPath !== "login") {
+	  		return true
+	  	} else {
+	  		return false
+	  	}
+	}),
+	isExperiments: Ember.computed('this.currentPath', function() {
+	  	if (this.currentPath === "experiments.index") {
+	  		return true
+	  	} else {
+	  		return false
+	  	}
+	}),
+	isParticipants: Ember.computed('this.currentPath', function() {
+	  	if (this.currentPath === "participants") {
+	  		return true
+	  	} else {
+	  		return false
+	  	}
+	}),
+	sizeContainer: function() {
```

Comment:
We should use bootstrap responsive utilities for this: http://getbootstrap.com/css/#responsive-utilities

There are three states here:
1. large screen, nav expanded
2. large screen, nav collapsed
3. small screen, nav collapsed (not expandable)